pkg/option/config.go

```diff
@@ -2591,13 +2598,37 @@ func (c *DaemonConfig) validateIPv6ClusterAllocCIDR() error {
 	return nil
 }
 
+func (c *DaemonConfig) validateIPv6NAT46x64CIDR() error {
+	ip, cidr, err := net.ParseCIDR(c.IPv6NAT46x64CIDR)
```

Comment:
optional nit: I understand this is copied over from `validateIPv6ClusterAllocCIDR`, but consider using `netip.ParsePrefix` for new code.